apps/meteor/server/features/EmailInbox/EmailInbox_Incoming.ts

```diff
@@ -161,12 +161,18 @@ export async function onEmailReceived(email: ParsedMail, inbox: string, departme
 		room = await QueueManager.unarchiveRoom(room);
 	}
 
-	let msg = email.text;
-
-	if (email.html) {
-		// Try to remove the signature and history
-		msg = stripHtml(email.html.replace(/<div name="messageSignatureSection.+/s, '')).result;
-	}
+	// TODO: html => md with turndown
+	const msg = email.html
+		? stripHtml(email.html, {
+				dumpLinkHrefsNearby: {
+					enabled: true,
+					putOnNewLine: false,
+					wrapHeads: '(',
+					wrapTails: ')',
+				},
+				skipHtmlDecoding: false,
+		  }).result
+		: email.text || '';
```

Comment:
How a parsed email looks like with this changes vs the old way?

Reply:
it looks closer to the actual e-mail, it was getting the text as default, which is no longer true for how e-mails are sent.

Reply:
Can you share a screenshot of what to expect?